client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -525,6 +545,15 @@ function returnModalCopy(
 				),
 			};
 		case 'domain':
+			if ( isRenewal ) {
+				return {
+					title: String( translate( 'You are about to remove your domain renewal from the cart' ) ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'You are about to remove your domain from the cart' )` *ES Score: 7*